src/types.rs

```diff
@@ -16,19 +16,22 @@ pub struct User {
 pub enum Backend {
     Unknown,
     FTD,
+    RAW,
 }
 
 impl Backend {
     pub fn accept(&self, path: &std::path::Path) -> bool {
         match self {
             Backend::FTD => path.extension() == Some(std::ffi::OsStr::new("ftd")),
+            Backend::RAW => true,
             Backend::Unknown => false,
         }
     }
 
     pub fn pattern(&self) -> Option<String> {
         match self {
             Backend::FTD => Some("**/*.ftd".to_string()),
+            Backend::RAW => Some("**/*.*".to_string()),
```

Comment:
In situations like this, where you know all the values would be hardcoded strings, instead of `Option<String>` you can also use `Option<&'static str>`.